src/views/ManageList.tsx

```diff
@@ -76,6 +85,44 @@ export function ManageList({ listPath }: Props) {
 		}
 	};
 
+	const handleInvite = async (
```

Comment:
Could all of this new logic be its own component that is rendered here instead? There's a problem developing in this file where 1. the component is doing too many things (re: [this other comment I left on PR #23](https://github.com/the-collab-lab/tcl-77-smart-shopping-list/pull/23#discussion_r1730064679)) and 2. now there's a function in this component called `handleSubmit` along side two different forms, making it hard to understand what that function is doing unless you read through its definition. This can be solved quickly by pushing this logic into its component and rendering it here instead 😄

Reply:
Yes please! I have an idea of what I want to do! I can work on this!

Reply:
Here are some screenshots of the component for your viewing pleasure!
![image](https://github.com/user-attachments/assets/a21cd3e7-57a2-4966-b76f-508e5911b419)
![image](https://github.com/user-attachments/assets/e165ff84-aa55-4d41-8d3a-54da26c7200d)

Reply:
WooooHoooo!!!!!